Pocket.Logger/Tests/ConfirmationLoggerTests.cs

```diff
@@ -140,7 +140,7 @@ public void Succeed_can_be_used_to_add_additional_properties_on_complete()
                .Properties
                .Select(_ => _.Value)
                .Should()
-               .ContainSingle(arg => arg == "bye!");
```

Comment:
All these tests are testing pointer equivalence, which happens to pass in this instance, but probably wasn't intended.